problems/[221211] 561. Array Partition.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * leetcode problem link: https://leetcode.com/problems/array-partition/
+ *
+ * @param {number[]} nums
+ * @return {number}
+ */
+var arrayPairSum = function (nums) {
+  const sortedNums = [...nums].sort((a, b) => a - b);
+
+  return sortedNums.reduce((sum, cur, index) => {
```

Comment:
```suggestion
  return [...nums].sort((a, b) => a - b).reduce((sum, cur, index) => {
```

별도의 sortedNums라는 변수를 만들 지 않고도 결과값을 도출할 수 있을 것 같아요